src/main/groovy/com/benjaminsproule/swagger/gradleplugin/classpath/ClassFinder.groovy

```diff
@@ -38,7 +43,7 @@ class ClassFinder {
         }
 
         Set<Class<?>> classes = new HashSet<Class<?>>()
-        ClassLoader classLoader = prepareClassLoader()
+        ClassLoader classLoader = classLoader
```

Comment:
Is this line required any more?

Reply:
No, not sure how I didn't spot that, removed.